docs/ChangeLog.md

```diff
@@ -1,5 +1,20 @@
 # Splunk Operator for Kubernetes Change Log
 
+## 2.0.0 (2022-05-27)
+
+* This is the 2.0.0 release. The Splunk Operator for Kubernetes is a supported platform for deploying Splunk Enterprise with the prerequisites and constraints laid out [here](https://github.com/splunk/splunk-operator/blob/master/docs/README.md#prerequisites-for-the-splunk-operator)
+
+* Operator SDK upgraded to version 1.21.0. Operator upgrade steps with necessary script updated [here](https://github.com/splunk/splunk-operator/blob/master/docs/SplunkOperatorUpgrade.md)
+
+* Introducing Monitoring Console CRD
+
+* Functional QA automation enhancements
+
+* Documentation Updates
+  
+* Changed to App Framework
```

Comment:
Can we move it on line #9

Reply:
`Changes to App Framework` or if we can add what changed in one line

Reply:
I think we also have set of Jira items to be added to the list?

Reply:
@sgontla can you add the CR names here, we can do that later also once this is merged to develop

Reply:
moved to top